### DISCUSSION TITLE
"Arrange an entire workspace of apps with just one shortcut." feature is missing one key feature (for me/our company)

### DISCUSSION THREAD

**Discussion by bramvanhoof** [rxhanson/RectanglePro-Community#188] [Ideas]
The "Arrange an entire workspace of apps with just one shortcut." is missing a key feature. 
For my workflow I am working with exceed, I work with multiple windows of the same application. 
Sometimes more, sometimes less windows are used.

For me it would be amazing if I could filter the window names e.g. all windows with the text "schematic" in the title go to that part of the screen on monitor 3, all windows with the text "layout" in the title go to that part of the screen on monitor 2, ...
Thanx for reading

**Comment by rxhanson** [comment]
Thanks for the feedback. With macOS being privacy oriented, window titles don't get passed to third party apps. It might be possible to obtain that information with screen recording permissions, but I actually haven't tried it out to see. I'll take a look next time I'm in a "new feature" sprint on Rectangle Pro.

**Comment by pltrant** [comment]
I would also love for this feature to exist.  I use multiple Chrome windows and want to position them into specific locations.

**Reply by jasonjwarren** [reply]
I'm hopeful this is, in fact, how the *App Layouts* work now.  I've got ~12 Chrome windows across 8 spaces, running at native 8k.  
🤞🏼 that after a restart these windows will be repopulated, with the sync'd tab groups where they should be.
